README.md

```diff
@@ -42,8 +42,7 @@ python -m uqcsbot
 
 ### Environment Variables
 
-* `DISCORD_BOT_TOKEN` for the Discord provided bot token.
-* `POSTGRES_URI_BOT` for the PostgreSQL connection string.
+See the [Tokens and Environment Variables wiki page](https://github.com/UQComputingSociety/uqcsbot-discord/wiki/Tokens-and-Environment-Variables) for what each environment variable is for and how to acquire tokens.
```

Comment:
MC hasn't been added yet (just putting this here so we don't forget)

Reply:
Done